serverless.yml

```diff
@@ -62,6 +62,7 @@ functions:
     environment:
       STATE_MACHINE_ARN: arn:aws:states:${self:provider.region}:#{AWS::AccountId}:stateMachine:aqts-capture-state-machine-${self:provider.stage}
       AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      OBJECT_SIZE_LIMIT: 150000000
```

Comment:
I'm not technically reviewing this, so feel free to ignore.

I don't believe this environment variable is being used in the handler. One or two lines of python need to be added in the for loop that starts at https://github.com/usgs/aqts-capture-trigger/blob/master/trigger/handler.py#L38.

Reply:
Here's the implementation from three months ago: https://github.com/usgs/aqts-capture-trigger/blob/841a04a43cb71595e9af2bbb185637726eff5eff/trigger/handler.py#L53

Reply:
ehh ... yes, thank you.